test/client/js/libs/handlebars/parse.js

```diff
@@ -312,6 +325,35 @@ describe("parse Handlebars helper", () => {
 		expect(actual).to.deep.equal(expected);
 	});
 
+	// Emoji
+	[{
+		name: "in text",
+		input: "Hello💬",
+		expected: 'Hello<span class="emoji">💬</span>',
+	}, {
+		name: "with modifiers",
+		input: "🤷‍♀️",
+		expected: '<span class="emoji">🤷‍♀️</span>',
+	}, {
+		name: "wrapped in style",
+		input: "Super \x034💚 green!",
+		expected: 'Super <span class="emoji"><span class="irc-fg4">💚</span></span><span class="irc-fg4"> green!</span>',
```

Comment:
This doesn't get optimized into a single `irc-fg4`? That's odd.

Reply:
Nope 😩
I'll add a FIXME comment and open an issue referring to this convo/test.